gammapy/maps/profile.py

```diff
@@ -48,6 +48,10 @@ class ImageProfileEstimator:
         smoothed = profile.smooth(kernel='gauss')
         smoothed.peek()
         plt.show()
+
```

Comment:
Again this example could be just changed to: `"$GAMMAPY_DATA/fermi-3fhl-gc/fermi-3fhl-gc-counts.fits.gz"`